src/main/java/com/box/sdk/BoxDeveloperEditionAPIConnection.java

```diff
@@ -226,7 +233,24 @@ String jwtConstructAssertion() {
         return assertion;
     }
 
-    PrivateKey decryptPrivateKey() {
+    private String getAlgorithmIdentifier() {
```

Comment:
I think this would be clearer as a `toString()` method on the enum itself. See https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxFile.java#L653-L707 as an example.